lib/watcher/status-wd.js

```diff
@@ -29,3 +29,49 @@ exports.master = function(handle) {
     handle.send(msg);
   });
 };
+
+// require() of strongloop.json from this file causes Trace Sequences to not
+// show up in arc Tracing for reasons unknown at this moment.
+//
+// This is a copy of strong-agent:lib/config.js with the offending requires
+// stubbed out.
+function configure(userKey, appName, options, env) {
+  /* eslint no-unused-vars:0, sort-vars:0, semi:0, no-empty:0 */
+  var home =
+          process.env.HOME || process.env.HOMEPATH || process.env.USERPROFILE;
+  var cwd = process.cwd()
+  var nfjson
+  var pkgjson
+  var userjson;
+
+  // Load configs from strongloop.json and package.json
+  try {
+    nfjson = {} // require(cwd + '/strongloop.json');
+  } catch (e) {
+    nfjson = {};
+  }
+  try {
+    pkgjson = require(cwd + '/package.json');
+  } catch (e) {
+    pkgjson = {};
+  }
+  try {
+    userjson = {} // require(home + '/strongloop.json');
+  } catch (e) {
+    userjson = {};
+  }
```

Comment:
I don't want to change anything, this is an almost literal copy of the original source